docs/documentation/cn/sql-reference/sql-functions/date-functions/date_format.md

```diff
@@ -0,0 +1,93 @@
+# date_format
```

Comment:
change directory date-functions to date-time-functions?

Reply:
Done